####### The service to start the battery charging

### Request

# start battery charging flag.

bool start

---

### Response
